### ISSUE TITLE
Suggestion: Validation ceremony changes

### ISSUE THREAD

**Issue by nightfallinme** [idena-network/idena-go#518]
i think there are some abusable flaws in validation ceremony and i believe they could be solved with some changes.

I think there should be 2 tier verification step, (not only short or long but instead status based tiers also). Current system allows abuse of bad behaviour like Canditate or Newbies to vote wrong (unstable consensus) or report wrong intentially, without risking anything (since they are just 1 invite away).  

instead of current system, first verified and human status could vote an in-house flip voting each other flips. then later everyone vote for newbie flips.

so basically , first verified and human vote flips from a pool chosen only of two and then all network vote only for newbie flips not verified/human flips. this way candidate and newbies have no chance to abuse intentially worng voting/reporting.
